test/automatedReleases.test.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const assert = require('assert');
+const sinon = require('sinon');
+const axios = require('axios');
+const {getChangelog} = require('../src/getChangelog');
+const { createRelease } = require('../src/createRelease');
+
+
+describe('Automated Releases', function() {
+    it('creates a draft release', async function() {
+        sinon.stub(axios, 'post').returns({status: 200});
```

Comment:
You should be stubbing `getChangelog()` and `createRelease()`, **not** `axios.post()`.

I think part of the confusion is that we have 2 separate `createRelease()` functions. Let's rename the function in `githubReleaseScript.js` to `createReleaseFromChangelog()`, and make that function call `getChangelog()` and `createRelease()`.

You should be calling `createReleaseFromChangelog()` in this test, and stubbing out `getChangelog()` and `createRelease()`

Reply:
Ok so the only problem I encounter doing this is that when the requests get stubbed they respond with a status code of 422 which if I remember correctly means the request is not authorized. Even though when I don't stub out the functions it works as intended.